src/instructlab/log.py

```diff
@@ -37,7 +37,7 @@ def isatty(self):
         return False
 
 
-def stdout_stderr_to_logger(logger, log_file):
+def stdout_stderr_to_logger(logger, log_file=None):
```

Comment:
I believe you made this change because a previous commit accidentally dropped some log file support that was added

Reply:
@leseb IIRC you added that support yes? Apologies for the ping if I am wrong

Reply:
will remove this, I think I fixed the broken commits that caused this to error out